experiments/conductor/cmd/runner/script_commands.go

```diff
@@ -0,0 +1,81 @@
+// Copyright 2025 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package runner
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"os"
+	"path/filepath"
+)
+
+// isExecutable checks if the given file path is executable.
+func isExecutable(info os.FileInfo) bool {
+	// Check if it's a regular file and has at least one execute permission bit set
+	mode := info.Mode()
+	return mode.IsRegular() && (mode.Perm()&0111 != 0)
+}
+
+// executeScript runs the script located at scriptPath
+func executeScript(opts *RunnerOptions, branch Branch) error {
+	ctx := context.TODO()
```

Comment:
Especially for a foot gun like this you need to set up the logging.
	close := setLoggingWriter(opts, branch)
	defer close()
	if branch.Command == "" {
		log.Printf("SKIPPING %s, no gcloud command", branch.Name)
		return
	}